### DISCUSSION TITLE
MockV3 Aggregator is not defined (already created MockV3Aggregator.sol and imported it)

### DISCUSSION THREAD

**Discussion by 0xwtfwtf** [Cyfrin/foundry-full-course-cu#1011] [General]
I finished the solidity course and am doing the python/brownie course, and working on the fund me section of it. I am getting an error that MockV3Aggregator is not defined even though I cut and paste the code into a file called MockV3Aggregator.sol and I imported it (see code below). This is from helpfulscripts.py. What else can i do to troubleshoot this?

I forgot to add that it's the MockV3Aggregator in deploy_mocks that is giving the error. It seems like it's being imported?

``` python
from brownie import network, config, accounts, MockV3Aggregator
from web3 import Web3

def get_account():
    if network.show_active() == "development":
        return accounts[0]
    else:
        return accounts.add(config["wallets"]["from_key"])
    
def deploy_mocks():
    if len(MockV3Aggregator) <= 0:
            #only need to deploy once
            Mock3Aggregator.deploy(18, Web3.toWei(2000, "ether"), {"from": account})

**Comment by 0xusmanf** [comment]
Not sure if this is the problem but there is a typo in the following line in your code.
```python
Mock3Aggregator.deploy(18, Web3.toWei(2000, "ether"), {"from": account})
```
It should be
```python
MockV3Aggregator.deploy(18, Web3.toWei(2000, "ether"), {"from": account})
```
